pipenv/core.py

```diff
@@ -1484,23 +1482,26 @@ def pip_install(
             ignore_hashes = False
     line = None
     # Try installing for each source in project.sources.
+    search_all_sources = project.settings.get("install_search_all_sources", False)
```

Comment:
Should we add public facing documentation for this option?

Reply:
Yes -- I am going to add that once we decide if this should be what its name is.

Reply:
I think the name `install_search_all_sources` is OK. Once you write the docs for this, you can figure if this works or not.

Reply:
Squeezing this into tonight's release -- let me know if there are any follow up tweaks you want to see to the docs or this PR.